config/default.config.yaml

```diff
@@ -66,7 +66,13 @@ middleware:
         # requests.
         picklePath: 'av_pickled_decryption.key'
 
-# Optional. A list of accepted mime type.
+# Optional. A list of accepted MIME types.
+# WARNING! It should be noted that this cannot be relied on for security:
+# it is possible to specially craft files that look like different kinds of file.
+# As one example, it is possible to craft a file which is both a PNG file and a
+# ZIP file yet will be detected as `image/png`.
+# Further, it should be generally assumed that most file formats have a way to
+# encode and attach extraneous data.
 #acceptedMimeType:
```

Comment:
I think this is much more readable with some whitespace added. I did change some wording too though.

```suggestion
# Optional. A list of accepted MIME types.
#
# WARNING! This cannot be relied on for security:
#
# it is possible to specially craft a file that looks like a different kind of file.
# As one example, it is possible for a file to be both a PNG file and a
# ZIP file yet will be detected as `image/png`.
# Further, it should be generally assumed that most file formats have a way to
# encode and attach extraneous data.
#
#acceptedMimeType:
```